console/my-account/concepts.mdx

```diff
@@ -25,6 +25,14 @@ categories:
 
 <Concept opened>
 
+## Active session
+
+An active session begins when a computer or device signs into your account. To connect you to your account, Scaleway uses JSON Web Tokens (JWT). JWT allows secure transmission of information between parties through tokens. These parties can be users, servers, or any other combination of services. In the case of Scaleway, when a user connects to their account (e.g. via an email address and password), we generate a token, associate it with the session and send it to the user. The token is composed of a header, a payload, and a signature. You can view a list of your active sessions, seeing all signed-in and recently active devices, in the [Scaleway console](https://console.scaleway.com/account/user/profile).
```

Comment:
I don't really understand what you mean by "seeing all signed-in", do you mean that you can view a list or your active sessions and see the things mentioned after?

Reply:
I mean that when you look at the list of your active-sessions, you see all the devices that are signed in and recently active (that "recently active" bit of the wording is from the console)

Which option do you prefer?

A) You can view a list of your active sessions in the Scaleway console, to see all signed-in and recently active devices
B) You can view a list of your active sessions in the Scaleway console
C) You can see the list of your active sessions in the Scaleway console, to get an overview of all devices that are signed in and recently active on your account

Reply:
soit A ou C pour moi

Reply:
Oui pareil soit A soit C !